extensions/arc/deployment/src/main/resources/dev-ui/qwc-arc-bean-graph.js

```diff
@@ -65,40 +65,48 @@ export class QwcArcBeanGraph extends LitElement {
         this._links = []
         this._nodes = []
         for (var l = 0; l < dependencyGraphsLinks.length; l++) {
-            let link = new Object();
-            link.source = dependencyGraphsNodes.findIndex(item => item.id === dependencyGraphsLinks[l].source);
```

Comment:
This was the problematic part - the node index value was derived from the `dependencyGraphsNodes` array but the graph data are stored in `this._nodes`. These two arrays have the same size but different ordering.

Reply:
Nice !